packages/inline-svg/index.js

```diff
@@ -1,3 +1,9 @@
-export default function InlineSvg({ src, ...props }) {
-  return <div dangerouslySetInnerHTML={{ __html: src }} {...props}></div>
+const supportedElements = {
+  div: 'div',
+  span: 'span',
+}
+
+export default function InlineSvg({ wrapper, src, ...props }) {
+  const Component = supportedElements[wrapper] || supportedElements['div']
```

Comment:
what happens if someone doesn't enter in `span` or `div` here?

Reply:
Currently only `div` and `span` are supported, if an element other than the `supportedElements` is provided, we fallback to `div`. Agreed we can make a more explicit contract using `spanWrapper` as a boolean.